tests/parsers/agilent_gen5/agilent_gen5_structure_test.py

```diff
@@ -17,18 +17,18 @@
 @pytest.mark.short
 def test_create_header_data_no_well_plate_id_in_filename() -> None:
     header_rows = [
-        "Software Version	3.12.08",
+        "Software Version   3.12.08",
```

Comment:
Similarly - I don't know what's happening with formatting here, but please fix before we merge